### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dkraeker**
Hope this helps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Wheel Ride](http://drawvr.com/wheel/) (by [@drawvr])
 - [Equalizer Ride](http://drawvr.com/equalizer/) (by [@drawvr])
 - [Test Scape](http://x627.com/testscape/) (by [Michael Verdi](http://x627.com/directory/))
+- [INFIVERSE](http://infiverse.com/) (by [@drawvr])
```
Comment (ngokevin) [inline README.md]: Can you add a short description? Thanks!

Comment (dkraeker) [inline README.md]: Infiverse= Collective of responsive 3D & WebVR websites made in MozVR A-Frame. Find cool experiences, templates, code snippets and more

File: README.md [hunk 2]
```diff
@@ -116,6 +116,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Wheel Ride](http://drawvr.com/wheel/) (by [@drawvr])
 - [Equalizer Ride](http://drawvr.com/equalizer/) (by [@drawvr])
 - [Test Scape](http://x627.com/testscape/) (by [Michael Verdi](http://x627.com/directory/))
+- [INFIVERSE](http://infiverse.com/) (by [@drawvr]) Infiverse= Collective of responsive 3D & WebVR websites made in MozVR A-Frame. Find cool experiences, templates, code snippets and more
```
Comment (ngokevin) [inline README.md]: Maybe:

```[INFIVERSE](http://infiverse.com/) - Navigate a collection of cool A-Frame sites by walking through rooms and doorways (by [@drawvr])```

Comment (dkraeker) [inline README.md]: sounds good

**Pull request merged**